funcx_endpoint/funcx_endpoint/executors/high_throughput/funcx_worker.py

```diff
@@ -146,24 +147,40 @@ def start(self):
                             len(serialized_result), self.result_size_limit
                         )
                 except Exception as e:
+                    exec_end = time.time()
                     log.exception(f"Caught an exception {e}")
                     result_package = {
                         "task_id": task_id,
                         "container_id": container_id,
                         "exception": self.serialize(
                             RemoteExceptionWrapper(*sys.exc_info())
                         ),
+                        "times": {
+                            "execution_start": exec_start,
+                            "execution_end": exec_end,
+                            "execution_time": exec_end - exec_start,
+                        },
                     }
                 else:
+                    exec_end = time.time()
                     log.debug("Execution completed without exception")
                     result_package = {
                         "task_id": task_id,
                         "container_id": container_id,
                         "result": serialized_result,
+                        "times": {
+                            "execution_start": exec_start,
+                            "execution_end": exec_end,
+                            "execution_time": exec_end - exec_start,
+                        },
```

Comment:
This whole block "feels like" try-finally.

I'd like to see it written
```python
exec_start = time.time()
try:
    ...
except ...:
    ...
else:
    ...
finally:
    exec_end = time.time()

exec_duration = exec_end - exec_start
result["times"] = {
    ...
}
```

That means there's never a branch where `exec_end` is missing, and the `"times"` block only needs to be written once.

Reply:
Good point. I'll fix it.